src/CookieStore.ts

```diff
@@ -101,13 +101,20 @@ class CookieStore {
     const persistedCookies = localStorage.getItem(PERSISTENCY_KEY)
 
     if (persistedCookies) {
-      const parsedCookies: [string, [string, any]] = JSON.parse(
-        persistedCookies
-      )
-
-      parsedCookies.forEach(([origin, cookie]) => {
-        this.store.set(origin, new Map(cookie))
-      })
+      try{
+        const parsedCookies: [string, [string, any]] = JSON.parse(
+          persistedCookies
+        )
+  
+        parsedCookies.forEach(([origin, cookie]) => {
+          this.store.set(origin, new Map(cookie))
+        })
+      }catch(_){
+        console.warn(`[MSW] the storage used for cookies has an invalid value "${localStorage.getItem(PERSISTENCY_KEY)}".
```

Comment:
What do you think if we slightly change this warning message?

```
[virtual-cookie] Failed to parse a stored cookie from the localStorage (key "{PERSISTENCY_KEY}").

Stored value:
${localStorage.getItem(PERSISTENCY_KET)}

Thrown exception:
${error}

Invalid value has been removed from localStorage to prevent subsequent failed parsing attempts.
```

Reply:
fine 😄